aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -132,6 +132,7 @@ pub enum FeatureFlag {
     TransactionSimulationEnhancement,
     CollectionOwner,
     EnableLoaderV2,
+    AccountAndCoinModuleEventMigration,
```

Comment:
could we call it Transaction instead of migration?

Reply:
Do you mean "translation" instead of migration?

Reply:
yeah, but why you want a separate migration flag for coin and account?